docker/README.md

```diff
@@ -0,0 +1,81 @@
+# Garage Docker
+
+Currently there's three versions of garage containers:
+  - ci: used for the continuous integration of garage in TravisCI and not
+    intended for final users.
+  - release: it's a published image based on the latest official release of
```

Comment:
this should just live in the release branch

Reply:
Once approved, I will back port this PR to the release branch. In the meantime, I have removed the image named rlworkgroup/garage:release.